mongo-oplog/index.d.ts

```diff
@@ -0,0 +1,19 @@
+// Type definitions for [cayasso/mongo-oplog]
```

Comment:
Use `// Type definitions for mongo-oplog 2.0`. (Assuming these types are for the most recent version of the package).